ehitchhiking-ui/src/app/pages/main-screen/trip-registration/trip-registration.component.ts

```diff
@@ -36,4 +41,22 @@ export class TripRegistrationComponent implements OnInit {
 		this.isShownViewListButton.emit(true);
 		this.isShownSaveRouteButton.emit(true);
 	}
+
+  isDriver() {
+    if (this.userState === UserState.driver) {
```

Comment:
why not just return this.userState === UserState.driver ?

Reply:
fixed